Modules/_bisectmodule.c

```diff
@@ -53,9 +53,14 @@ bisect_right(PyObject *self, PyObject *args, PyObject *kw)
     Py_ssize_t index;
     static char *keywords[] = {"a", "x", "lo", "hi", NULL};
 
-    if (!PyArg_ParseTupleAndKeywords(args, kw, "OO|nn:bisect_right",
-        keywords, &list, &item, &lo, &hi))
-        return NULL;
+    if (kw==NULL && PyTuple_GET_SIZE(args)==2) {
+        list = PyTuple_GET_ITEM(args, 0);
+        item = PyTuple_GET_ITEM(args, 1);
+    } else {
```

Comment:
For PEP 7 `}` and `else {` should be on separate lines.
```c
    }
    else {
```

Reply:
I see that in PEP 7 but it looks weird to me.  I believe we historically allowed "} else {" in multiple places in the code when that reflects how the author thinks about the code.  If you insist, I'll change it, but I like it better as-is.